src/test/regress/sql/multi_partitioning.sql

```diff
@@ -986,16 +986,21 @@ ORDER BY
 COMMIT;
 
 -- test partition-wise join
-
 CREATE TABLE partitioning_hash_join_test(id int, subid int) PARTITION BY HASH(subid);
 CREATE TABLE partitioning_hash_join_test_0 PARTITION OF partitioning_hash_join_test FOR VALUES WITH (MODULUS 3, REMAINDER 0);
 CREATE TABLE partitioning_hash_join_test_1 PARTITION OF partitioning_hash_join_test FOR VALUES WITH (MODULUS 3, REMAINDER 1);
 CREATE TABLE partitioning_hash_join_test_2 PARTITION OF partitioning_hash_join_test FOR VALUES WITH (MODULUS 3, REMAINDER 2);
 
 SELECT create_distributed_table('partitioning_hash_join_test', 'id');
 
--- see the query plan without partition-wise join
-EXPLAIN
+SELECT success FROM run_command_on_workers('alter system set enable_mergejoin to off');
```

Comment:
These are really limiting postgres planner's options.

Reply:
That's my goal 😃

Reply:
I mean it's not like we're doing this in our code, just in the test… seems fine to me.